kairon/api/app/routers/bot/bot.py

```
@@ -1658,3 +1660,14 @@ async def get_slot_actions(
     llm_models = MongoProcessor.get_slot_mapped_actions(current_user.get_bot(), slot_name)
     return Response(data=llm_models)
 
+
+
+@router.get("/mail_channel/logs", response_model=Response)
+async def get_action_server_logs(start_idx: int = 0, page_size: int = 10,
```

Ruff (0.8.0)

Redefinition of unused `get_action_server_logs` from line 826

(F811)